conda/recipes/dask-cuda/meta.yaml

```diff
@@ -24,11 +24,9 @@ requirements:
     - setuptools
   run:
     - python
-    - dask >=2.4.0
```

Comment:
It's worth noting this is older than the version in `requirements.txt`. Would suggest we go with the newer of the two (otherwise no change need). Just mentioning this for awareness